apps/frontend/src/libs/hooks/use-search/use-search.hook.ts

```diff
@@ -13,14 +14,20 @@ const useSearch = (): {
 
 	useEffect(() => {
 		const updatedSearchParameters = new URLSearchParams(searchParameters);
-
 		updatedSearchParameters.set(searchParameterName, search);
-
 		setSearchParameters(updatedSearchParameters);
 	}, [search, searchParameters, searchParameterName, setSearchParameters]);
 
+	const resetSearch = (): void => {
+		setSearch("");
+		const updatedSearchParameters = new URLSearchParams(searchParameters);
+		updatedSearchParameters.delete(searchParameterName);
+		setSearchParameters(updatedSearchParameters);
+	};
+
```

Comment:
Not a the best idea to keep this in use-search hook. There is a reset method in useForm, I think we should use it

Reply:
On first thought I agree that useForm's reset is the right way to handle resetting the form fields. However, useSearch is also managing the URL query parameters, which resetForm can't handle on its own. To fully clear the search, we need both: resetFor for the input and resetSearch in useSearch to remove the query parameter from the URL. 

Should i just create a new search hook for Modals or should i rewrite the useSearch hook to be able to handle both?

Reply:
Shouldn't the query parameters update be triggered by a search change? As I see from the logic when the value inside the Search component is updated, onSearch (from useSearch) is triggered and it updates the query params value in useEffect, doesn't it?

Reply:
I see what you're saying about useForm's reset. Just to clarify, are you suggesting that I manage the search input directly within useForm and update the query parameters through the form's state, rather than using a separate useSearch hook for this? That way, resetForm would handle both the input and query parameter reset together.

Reply:
yes, I see what you mean, i will get this sorted asap

Reply:
I have simplified it, and made use of the `UseFormReset `instead. I think this is cleaner and more reusable approach.